mods/sethome/init.lua

```diff
@@ -2,64 +2,59 @@ local homes_file = minetest.get_worldpath() .. "/homes"
 local homepos = {}
 
 local function loadhomes()
-    local input = io.open(homes_file, "r")
-    if input then
+	local input = io.open(homes_file, "r")
+	if input then
 		repeat
-            local x = input:read("*n")
-            if x == nil then
-            	break
-            end
-            local y = input:read("*n")
-            local z = input:read("*n")
-            local name = input:read("*l")
-            homepos[name:sub(2)] = {x = x, y = y, z = z}
-        until input:read(0) == nil
-        io.close(input)
-    else
-        homepos = {}
-    end
+			local x = input:read("*n")
```

Comment:
t4im, i tried that bit of code with no luck btw, but if its a tas bit off well idk im not good with strings yet :)